examples/plot_simulate_gamma.py

```diff
@@ -44,44 +54,35 @@
 dpls = simulate_dipole(net)
 
 ###############################################################################
-# The network requires some time to reach steady state. Hence, we omit the
-# first 50 ms in our time-frequency analysis.
+# Take a look at how different cell types respond to the exogenous drive. Note
+# the periodic firing pattern of all cell types. While the basket cells fire
+# relatively synchronously, the pyramidal cell populations display a more
+# varied pattern, in which only a fraction of cells reach firing threshold.
+net.cell_response.plot_spikes_raster()
+
+###############################################################################
+# To confirm that the periodicity observed in the firing patterns correspond to
+# a population oscillation in the gamma-range, we can plot the time-frequency
+# representation together with the signal. Note that the network requires some
+# time to reach steady state. Hence, we omit the first 50 ms in our analysis.
 
-tstart = 50
+tmin = 50
 trial_idx = 0  # pick first trial
-mask = dpls[trial_idx].times > tstart
-times = dpls[trial_idx].times[mask]
-data = dpls[trial_idx].data['agg'][mask]
+decim = 8  # decimate plots by a factor of 8 (from 40 to 5 kHz)
 
-###############################################################################
-# We can plot the time-frequency response using MNE
-import numpy as np
+# plot dipole time course and time-frequency representation in same figure
 import matplotlib.pyplot as plt
-from mne.time_frequency import tfr_array_morlet
```

Comment:
so now the issue is that the user will be surprised with an import warning since the installation instructions didn't talk about installing MNE and neither does the example.